scripts/migrate_operations_to_workflows.py

```
@@ -0,0 +1,277 @@
+            
+            # Verify tables exist
+            self.source_table.load()
+            self.target_table.load()
+            
+            logger.info(f"Source table: {self.source_table_name}")
+            logger.info(f"Target table: {self.target_table_name}")
+            logger.info(f"Dry run mode: {self.dry_run}")
+            
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'ResourceNotFoundException':
+                logger.error(f"Table not found: {e.response['Error']['Message']}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)